src/pages/components/leadspace.mdx

```diff
@@ -70,23 +88,28 @@ For detailed guidelines around usage, also reference the [Lead space pattern pag
 
 There are two color themes to choose from when using Lead space:
 
-- Use `Dark (gray-100)` when the background image has many shadows, is dimly lit, or uses a dark color palette. The gradient overlay will render dark and the text will be white.
-- Use `Light (white)` when the background image is well-lit or brightly colored. The gradient overlay will render white and the text will be gray-100.
+### Dark theme
 
-![Dark theme](../../images/pattern/leadspace/LeadSpace_ColorThemes_Dark.jpg)
+Use `Dark (gray-100)` when the background image has many shadows, is dimly lit, or uses a dark color palette. The gradient overlay will render dark and the text will be white.
 
-<Caption>Dark</Caption>
+![Dark theme](../../images/pattern/leadspace/leadspace_colortheme_dark_demo.jpg)
+
+<Caption>Dark Lead space – Super – 640px height at xlg 1312px breakpoint</Caption>
 <br />
 
-![Light theme](../../images/pattern/leadspace/LeadSpace_ColorThemes_Light.jpg)
+### Light theme
+
+Use `Light (white)` when the background image is well-lit or brightly colored. The gradient overlay will render white and the text will be gray-100.
 
-<Caption>Light</Caption>
+![Light_theme](../../images/pattern/leadspace/leadspace_colortheme_light_demo.jpg)
 
-## Left-aligned style options
+<Caption>Light Lead space – Super – 640px height at xlg 1312px break point</Caption>
 
-### Type styles
+## Type styles
 
-There are two expressive type styles to choose from when using Lead space left-aligned:
+### Lead space left-aligned
+
+There are two recommended expressive type styles to choose from when using Lead space left-aligned:
```

Comment:
Thanks for adding this though I was hoping we can...one day...narrow down to 1 type style per height. But for now, totally accurate.